chalk-solve/src/clauses.rs

```diff
@@ -164,6 +164,10 @@ pub fn push_auto_trait_impls<I: Interner>(
             push_auto_trait_impls_generator_witness(builder, auto_trait_id, generator_id);
         }
 
+        // Unimplemented
+        TypeName::OpaqueType(_) => {}
```

Comment:
Would it be cleaner to replace the `unimplemented!` calls in the `constituent_types` fn above with a `FIXME` comment and a reference to some issue? I'm not familiar with the chalk style guide, but I feel like leaving in the unimplemented calls could lead to them accidentally being triggered in the future.

Something like `// FIXME (issue#12345) implement these`

Reply:
The problem is that in order to do that, we would have to change the function signature to an `Option<...>`. If this was a long term solution, yeah that might be cleaner. But for now, this is fine to reduce churn.